jsonschema-core.xml

```diff
@@ -2826,7 +2826,8 @@ https://example.com/schemas/common#/$defs/count/minimum
                         be included if the validation was successful.  The value
                         for this property is an object where the keys are the names of
                         keywords and the values are the error message produced by the
-                        associated keyword.
+                        associated keyword.  If the subschema itself is producing the
+                        error, that error MUST be listed with an empty string key.
```

Comment:
What kind of errors do subschemas produce on their own?

Reply:
Maybe this can be more explicit, but something needs to be in place to handle the `false` schema.  For example:

```json
{
  "properties": {
    "foo": false
  }
}
```

You still need an error message for the `foo` property when it's included in the instance.  With the proposed format it'd be:

```json
{
   "valid": false,
   "nested": [
    {
      "valid": false,
      "instanceLocation": "/foo",
      "schemaLocation": "http://example/schema#/properties/foo"
      "evaluationPath": "/properties/foo",
      "errors": {
        "": "All values fail against the false schema"   // this
      }
    }
  ]
}
```

There's not a keyword that produces the error.  While someone technically _could_, I expect no one would actually create an empty-string keyword, so the empty string is sufficient to declare that this error came from the schema.

This is the immediate example I can think of.  I'm sure other people can conceive of other reasons to have a schema produce an error directly.  I worded it this way because I didn't want to limit to the `false` schema case.

Reply:
@gregsdennis thanks, this makes sense.  I'm a little skeptical of other use cases but that's not a reason to discourage, much less forbid them.  How would you feel about noting that the boolean `false` schema is the most common (or some vaguely similar wording) use case for this?  I feel like it would make it more clear why this has to be there.

I think something that'd probably want to avoid would be some sort of automatic roll-up error that appears in every failed dynamic scope / output unit.  That was the sort of thing that had me concerned- where errors are repeated without really adding anything.

Reply:
Sure I can add a note to that effect.